include/inset_state.hpp

```diff
@@ -166,19 +193,21 @@ class InsetState
     char,
     unsigned int) const;
   bool is_input_target_area_missing(const std::string &) const;
+  void is_simple(const char* caller_func);
```

Comment:
This is very useful enhancement.

Reply:
Haha, thank you! Agreed, more debugging info to see exactly where things went wrong.